packages/mobile/src/backup/utils.ts

```diff
@@ -12,7 +12,10 @@ export const MNEMONIC_SPLITTER = 'celo'
 export async function createQuizWordList(mnemonic: string, language: string | null) {
   const disallowedWordSet = new Set(mnemonic.split(' '))
   const languageWordList = getWordlist(language)
-  const wordOptions: string = await generateMnemonic(1000, null, languageWordList)
+  const wordOptions: string = await generateMnemonic(
+    MnemonicStrength.s256_24words,
+    languageWordList
+  )
```

Comment:
We should be passing in RN-bip39 here right?

Reply:
Yes. Nice catch 👍